python/orchestrate/_internal/http_handler.py

```diff
@@ -1,8 +1,15 @@
-import os
 import json
+import os
+from dataclasses import dataclass
 from typing import Any, Mapping, Optional
+from xml.etree import ElementTree
```

Comment:
I think dependabot takes a dim view of xml.etree and maybe the defusedxml library works just as well here?

Reply:
Hm. I'm not sure what you mean here. Is there a security advisory out on the stdlib?

Reply:
Well, that's quite a conundrum. Introduce a third party dependency to keep dependabot at bay or keep getting annoyed at dependabot telling us that the XML data we're parsing is trusted?

Maybe what I should do instead here is just to return the full response text, unparsed?